[1/10] Linking C static library esp-idf/main/libmain.a
[2/10] Performing build step for 'bootloader'
[1/1] cd /home/ivanool/Escritorio/KANA/TOHA/build/bootloader/esp-idf/esptool_py && /home/ivanool/.espressif/python_env/idf5.2_py3.12_env/bin/python /home/ivanool/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 bootloader 0x1000 /home/ivanool/Escritorio/KANA/TOHA/build/bootloader/bootloader.bin
Bootloader binary size 0x6870 bytes. 0x790 bytes (7%) free.
[3/10] No install step for 'bootloader'
[4/10] Completed 'bootloader'
[5/10] cd /home/ivanool/Escritorio/KANA/TOHA/build/esp-idf/ixora && /home/ivanool/.espressif/python_env/idf5.2_py3.12_env/bin/python /home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py 0xb0000 /home/ivanool/Escritorio/KANA/TOHA/spiffs_image /home/ivanool/Escritorio/KANA/TOHA/build/storage.bin --page-size=1024 --obj-name-len=256 --meta-len=4 --use-magic --use-magic-len
FAILED: esp-idf/ixora/CMakeFiles/spiffs_storage_bin /home/ivanool/Escritorio/KANA/TOHA/build/esp-idf/ixora/CMakeFiles/spiffs_storage_bin 
cd /home/ivanool/Escritorio/KANA/TOHA/build/esp-idf/ixora && /home/ivanool/.espressif/python_env/idf5.2_py3.12_env/bin/python /home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py 0xb0000 /home/ivanool/Escritorio/KANA/TOHA/spiffs_image /home/ivanool/Escritorio/KANA/TOHA/build/storage.bin --page-size=1024 --obj-name-len=256 --meta-len=4 --use-magic --use-magic-len
Traceback (most recent call last):
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 434, in create_file
    block.update_obj(contents_chunk)
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 349, in update_obj
    raise SpiffsFullError()
SpiffsFullError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 438, in create_file
    raise SpiffsFullError
SpiffsFullError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 593, in <module>
    main()
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 585, in main
    spiffs.create_file('/' + os.path.relpath(full_path, args.base_dir).replace('\\', '/'), full_path)
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 449, in create_file
    block = self._create_block()
            ^^^^^^^^^^^^^^^^^^^^
  File "/home/ivanool/esp/esp-idf/components/spiffs/spiffsgen.py", line 398, in _create_block
    raise SpiffsFullError('the image size has been exceeded')
SpiffsFullError: the image size has been exceeded
[6/10] Generating ld/sections.ld
ninja: build stopped: subcommand failed.
